internal/util/labels_annotations.go

```diff
@@ -24,6 +24,7 @@ const (
 	CSIDeleteSnapshotSecretName      = "velero.io/csi-deletesnapshotsecret-name"
 	CSIDeleteSnapshotSecretNamespace = "velero.io/csi-deletesnapshotsecret-namespace"
 	CSIVSCDeletionPolicy             = "velero.io/csi-vsc-deletion-policy"
+	VolumeSnapshotClassSelectorLabel = "velero.io/csi-volumesnapshot-class"
```

Comment:
In the PR's description, you say this label needs to be applied. Is this label applied by the administrator/user?

Reply:
Yes, the administrator/user is expected to label the volumesnapshotclass that they want Velero to use for snapshotting CSI volumes for a driver.

Reply:
Need to update the docs with this docs. I can think of https://velero.io/docs/csi/ anywhere else you can think of?

Reply:
I think that's the only place. Thanks!